modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/bookmark/BookmarkHelper.kt

```diff
@@ -35,7 +37,10 @@ class BookmarkHelper @Inject constructor(
         context: Context,
     ) {
         if (!shouldAllowAddBookmark()) return
-        if (context.isAppForeground()) {
+        if (context.isAppForeground() &&
+            Util.getAppPlatform(context) == AppPlatform.Phone &&
+            !Util.isCarUiMode(context)
```

Comment:
> Util.isCarUiMode(context) is returning false for me even when my phone is connected to a DHU. This might just be something weird about how the DHU works, I'm not sure. 

That was my assumption as well. I tried replacing it with Car Connection API [here](https://github.com/Automattic/pocket-casts-android/compare/task/bookmark-creation-sound...task/android-auto-add-bookmark?expand=1), and this works with DHU. Can you give it a try before I create a pull request?

Reply:
👍 That worked well in my testing both with the DHU and with my wife's car.

Reply:
Thanks for confirming! I pushed the changes in https://github.com/Automattic/pocket-casts-android/pull/1433/commits/173e028444c483960762ca9962051c9adc19a151 in case you have any code-related review comments.